### ISSUE TITLE
mouse and keyboard controller config

### ISSUE THREAD

**Issue by MrRokiIsBored** [Vita3K/Vita3K#2857]
i always read that you can map psvita controls with the mouse and keyboard but how and if there is no way to do that, can you make mouse controls support cause its not easy to play the game with keyboard only

i would really love that, other than that the vita3k project is the most impressive psvita emulator i saw in my life, without this project, i could have seen half of the games i played go with the dust and gets forgotten

**Comment by Folko-Ven** [comment]
Double it! Please make it possible to control it with a mouse.

**Comment by Zangetsu38** [comment]
I no understund want you want here ?
Mouse is only for touch screen ?

**Comment by Folko-Ven** [comment]
> 

The ability to control the gamepad stick with the mouse. Now this is only possible using the keyboard (keys i j k l)

**Comment by Zangetsu38** [comment]
> > 
> 
> The ability to control the gamepad stick with the mouse. Now this is only possible using the keyboard (keys i j k l) For example, yuzu provides such functionality

I keep no understund have no sensé, mouse is only here for Touch screen that all

**Comment by Folko-Ven** [comment]
> 

I'm asking to add the ability to control the stick through mouse movement, I don't know how else to explain it. Many games are much more convenient to control this way (for example, the legend of zelda tears of the kingdom or Metroid Prime Remastered). Yuzu implemented a keyboard shortcut that intercepts the mouse and uses it to control the gamepad stick.

**Comment by Vinddictive** [comment]
we still haven't had this?

**Comment by Folko-Ven** [comment]
No, I waited until the game I wanted to play (Freedom Wars) was ported to PC.
